spec.html

```diff
@@ -6415,7 +6437,7 @@ <h1>
         <dd>It is used to create a List value whose elements are provided by the indexed properties of _obj_. _elementTypes_ contains the names of ECMAScript Language Types that are allowed for element values of the List that is created.</dd>
       </dl>
       <emu-alg>
-        1. If _elementTypes_ is not present, set _elementTypes_ to « Undefined, Null, Boolean, String, Symbol, Number, BigInt, Object ».
+        1. If _elementTypes_ is not present, set _elementTypes_ to « ~undefined~, ~null~, ~boolean~, ~string~, ~symbol~, ~number~, ~bigint~, ~object~ ».
```

Comment:
Presumably the signature of this AO needs updating, or alternatively we could `<dfn>` "names of ECMAScript Language Types".

Reply:
I think we don't need this kind of generality anyway. `elementTypes` only ever has two values. Maybe we replace it with an enum?

Reply:
Temporal calls it with `« Object »`, though maybe that's one of the ones you're referring to?

I think making this more specific would actually make it _more_ verbose, so I'm inclined to stick with the general version, but not strongly.